zcash_client_sqlite/src/wallet/db.rs

```diff
@@ -1074,7 +1075,10 @@ WITH unioned AS (
            a.diversifier_index_be       AS diversifier_index_be,
            ro.value                     AS value,
            ro.is_change                 AS is_change,
-           ro.memo                      AS memo
+           ro.memo                      AS memo,
+           IFNULL(t.trust_status, 0)    AS is_trusted,
```

Comment:
Possible bug: the new `is_trusted` column of `unioned` is never used or exposed in `v_tx_outputs`.